test/images/webhook/pods.go

```diff
@@ -101,3 +101,41 @@ func mutatePods(ar v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 	}
 	return &reviewResponse
 }
+
+// denyAttachingPods denies `kubectl attach to-be-attached-pod -i -c=container1"
+// or equivalent client requests.
+func denyAttachingPods(ar v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
```

Comment:
denyAttachingToPods is slightly more understandable.

Reply:
Actually it appears to deny attaching to stdin in one particular container on one particular pod? How about "denySpecificAttachment"?